packer/centos7-katello-devel-stable.json

```diff
@@ -24,8 +24,8 @@
             "boot_wait": "2s",
             "shutdown_command": "echo 'vagrant' | sudo -S /sbin/halt -h -p",
             "qemuargs": [
-                [ "-m", "8192M" ],
-                [ "-smp", "2" ]
+                [ "-m", "12G" ],
+                [ "-smp", "4" ]
```

Comment:
This is saying "use up to 4 cores", so it won't error when there are less resources available. [reference](https://wiki.gentoo.org/wiki/QEMU/Options#Processor)